prog/install_rhizome.rb

```diff
@@ -7,6 +7,8 @@
 class Prog::InstallRhizome < Prog::Base
   subject_is :sshable
 
+  SKIP_VALIDATION = ["Gemfile.lock"]
```

Comment:
What's changed in the `Gemfile.lock`? If it's changed, there might be something missing in our lock file

Reply:
We don't have a unified Gemfile for the whole Rhizome. Instead, we have one in rhizome/common, one in rhizome/host. Therefore, the deployed Gemfile.lock in the sshable can be different than what we have in the repo. The Gemfile.lock in the repo is valid for the VmHost, but not the PG VM for example.